src/user-event/press/__tests__/press.real-timers.test.tsx

```diff
@@ -198,7 +198,7 @@ describe('userEvent.press with real timers', () => {
     );
     await userEvent.press(screen.getByText('press me'));
 
-    expect(getEventsNames(events)).toEqual(['pressIn', 'press', 'pressOut']);
+    expect(getEventsNames(events)).toEqual(['pressIn', 'pressOut', 'press']);
```

Comment:
there does not seem to be a similar test for the long press event. it would be nice to have one since a similar bug of order of events could occur with longPress